psh/test_ls.py

```diff
@@ -39,7 +39,23 @@ def prepare_testdir(p, files):
     psh.assert_cmd(p, f'mkdir {TEST_DIR_BASIC}/dir', '', msg)
 
 
-def create_pattern(files, separate_lines=False):
+def create_soft_pattern(files):
```

Comment:
`create_soft_pattern()` and `create_strict_pattern()` can be joined into one function with one parameter like 'ordering=strict/soft`. Please add more info in docstring about this option purpose.

Reply:
I would like to leave these 2 functions here. I tried to do it in `create_pattern()`, but the result was `spaghetti code` with nested if's. I think that used approach is much more readable.